core/rfb.js

```diff
@@ -2323,6 +2346,20 @@ export default class RFB extends EventTargetMixin {
         return this._fail("Unexpected SetColorMapEntries message");
     }
 
+    _writeClipboard(text) {
+        if (this._viewOnly) return;
+        (async () => {
+            if (await isAsyncClipboardAvailable()) {
+                await this._clipboard.writeClipboard(text);
+            } else {
+                // Dispatch event for the alternative clipboard
+                this.dispatchEvent(
+                    new CustomEvent("clipboard", {detail: {text: text}})
```

Comment:
The API documentation should be updated to reflect that this is now only used sometimes.